src/main/java/spim/fiji/plugin/Image_Fusion.java

```diff
@@ -35,13 +35,13 @@
 
 public class Image_Fusion implements PlugIn
 {
-	public static ArrayList< Fusion > staticFusionAlgorithms = new ArrayList< Fusion >();
+	public final static ArrayList< Fusion > staticFusionAlgorithms = new ArrayList< Fusion >();
```

Comment:
Aj, my bad!  I have to admit that the Java convention of considering everything to be a public api unless declared as "internal" to be very confusing (it makes much more sense to me to consider everything "internal" unless exposed as an external api).  If anyone else uses this code, please do revert my change (and remove final).